### DISCUSSION TITLE
Open a PR to resolve issues #128 on Community Website

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#1045] [First-Timers]
It would be great if someone can open a PR for this. :)

### What to do:
Update the community website's screenshot on README with a new updated screenshot
🟩 

### Issue:
https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/issues/128

### Old Screenshot in README:
![outdated screenshot](https://user-images.githubusercontent.com/61991582/92185956-22aee080-ee4d-11ea-9553-8598c74edde5.png)

### Current site:
https://eddiehub.org/

Issue opened by- [Prathamkrishna](https://github.com/Prathamkrishna)

**Comment by BOLT04** [comment]
Hi @Vyvy-vi, I noticed in your screenshot the icons for GitHub and the Website of each moderator doesn't work. Maybe we could open an issue to fix that too 😃?
It might a browser that it's not supporting or having issues, but we could still open an issue for it 👍
